cmd/autoheal/main.go

```diff
@@ -33,6 +33,7 @@ var rootCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(serverCmd)
+	flag.Set("logtostderr", "true")
```

Comment:
This seems to be the way the k8s vendor code does it. A quick search for `logtostderr` will show this line in multiple places.